src/component/NodeEditor/NodeEditor.tsx

```diff
@@ -1208,9 +1208,9 @@ const mapStateToProps = ({
     language,
     nodeEditorOpen,
     actionToEdit,
-    localizations,
     definition,
```

Comment:
thinking we could use `localizations` state for `cleanUpLocalizations` and the update router methods, and lose `definition` state as a dependency; what do you think? my bias being to not read from `definition` state if we can help it